packages/api/tests/performance/lib/test-write-granules.js

```diff
@@ -0,0 +1,264 @@
+'use strict';
+
+const test = require('ava');
+const pSettle = require('p-settle');
+
+const cryptoRandomString = require('crypto-random-string');
+const cloneDeep = require('lodash/cloneDeep');
+
+const {
+  getEsClient,
+  Search,
+} = require('@cumulus/es-client/search');
+const { createSnsTopic } = require('@cumulus/aws-client/SNS');
+const StepFunctions = require('@cumulus/aws-client/StepFunctions');
+
+const { constructCollectionId } = require('@cumulus/message/Collections');
+const {
+  fakeCollectionRecordFactory,
+  fakeExecutionRecordFactory,
+  fakeProviderRecordFactory,
+  fakePdrRecordFactory,
+  generateLocalTestDb,
+  destroyLocalTestDb,
+  CollectionPgModel,
+  ProviderPgModel,
+  ExecutionPgModel,
+  GranulesExecutionsPgModel,
+  GranulePgModel,
+  FilePgModel,
+  PdrPgModel,
+  migrationDir,
+  TableNames,
+} = require('@cumulus/db');
+const { sns, sqs } = require('@cumulus/aws-client/services');
+const {
+  SubscribeCommand,
+  DeleteTopicCommand,
+} = require('@aws-sdk/client-sns');
+const {
+  getExecutionUrlFromArn,
+} = require('@cumulus/message/Executions');
+const {
+  createTestIndex,
+  cleanupTestIndex,
+} = require('@cumulus/es-client/testUtils');
+
+const {
+  writeGranulesFromMessage,
+} = require('../../../lib/writeRecords/write-granules');
+const { fakeFileFactory, fakeGranuleFactoryV2 } = require('../../../lib/testUtils');
+
+test.before(async (t) => {
+  // Test configuration values
+  t.context.concurrency = 100;
+  process.env.maxDbPool = 100;
```

Comment:
here maxDbPool is being set as an env variable where elsewhere it seems to be expected as an argument to a function

Reply:
Both are possible, depending on the function - without knowing which function we're talking about, I can't comment on the difference, but using env is one of the accepted ways of handling DB configuration in prod/test, generically. 

 That said, it's a fair point that this is working *because* we don't provide a default as part of the test environment configuration - that's deliberate and a convention, but explicit > implicit.      I'm fine making the update to pass it directly, particularly as this likely impacts, in theory, the admin connection. 
 
 👍🏻

Reply:
Should it be `env.dbMaxPool`? 
https://github.com/nasa/cumulus/blob/v18.5.1/packages/db/src/config.ts#L200

Reply:
consider rename all maxDbPool to existing naming dbMaxPool

Reply:
Excellent point.     I'm updating the logging to make this more clear and demonstrate what *knex* thinks the setting is.    Good catch, both of you.

Reply:
cb98efaa465